tools/verify_source_code.sh

```diff
@@ -24,13 +24,10 @@ SCRIPTDIR="$(cd $(dirname "$0")/ && pwd)"
 source "$SCRIPTDIR/load_config.sh" $1
 PARENTDIR="$(dirname "$SCRIPTDIR")"
 
-REPORT_DIR=$PARENTDIR/report
-REPORT_FILE=$REPORT_DIR/apache-rat-report.txt
-
-mkdir -p $REPORT_DIR
-touch $REPORT_FILE
-
-java -jar $SCRIPTDIR/lib/apache-rat-0.13-SNAPSHOT.jar -a $OPENWHISK_CLEANED_SOURCE_DIR > $REPORT_FILE
+# run Apache rat to check headers
+cd $OPENWHISK_CLEANED_SOURCE_DIR
+cp $SCRIPTDIR/lib/pom.xml ./
+mvn clean apache-rat:check
```

Comment:
I think it is good to change line 38 of this file
cd $OPENWHISK_CLEANED_SOURCE_DIR/$repo_name && ls {LICENSE*,NOTICE*}
into 
cd $OPENWHISK_SOURCE_DIR/$repo_name && ls {LICENSE*,NOTICE*}

because we do not need to package the source before running verify_source_code.sh.

Rebase ur code with the latest master. Besides the above change, it looks good to me.